app/views/groups/_add_member_modal.html.erb

```diff
@@ -22,13 +22,39 @@
           </div>
           <div class="form-group">
             <label for="emails">Email IDs:</label>
-            <textarea class="form-control form-input" rows="5" id="emails" name="group_member[emails]"></textarea>
+            <textarea class="form-control form-input" rows="4" id="emails" name="group_member[emails]"></textarea>
           </div>
           <div class="modal-footer">
             <%= form.submit "Add members", class: 'btn primary-button' %>
           </div>
         <% end %>
       </div>
+      <div class="mx-auto text-center p-3">
+        <p>OR USE</p>
+        <a class="btn primary-button" data-toggle="collapse" href="#collapseExample" aria-expanded="false" aria-controls="collapseExample" onclick="linkGenerate()">
+          <i class="fas fa-external-link-alt"></i>
+          <span>Group Invite Link</span>
+        </a>
+        <div class="collapse" id="collapseExample">
+          <textarea id="embedInviteLink" class="group-invite-link-text" rows="4" cols="35"><%= group_url(@group) %>/invite/<%= @group.group_token %></textarea>
+        </div>
+      </div>
     </div>
   </div>
 </div>
+
+<script>
+   function linkGenerate() {
```

Comment:
use  rails remote form http://www.korenlc.com/remote-true-in-rails-forms/

Reply:
using remote:true for ajax request